.circleci/run-firesim-scala-tests.sh

```diff
@@ -13,13 +13,20 @@ source $SCRIPT_DIR/defaults.sh
 # call clean on exit
 trap clean EXIT
 
+# Directory locations for handling firesim-local installations of libelf/libdwarf
+# This would generally be handled by build-setup.sh/firesim-setup.sh
+firesim_sysroot=lib-install
```

Comment:
Might be safer to make this a absolute path? `$LOCAL_CHIPYARD_DIR/sims/firesim/lib-install`

Reply:
This is done in the next two lines? What's your concern?

Reply:
oh, misread this.